modules/nlu/src/views/intents/index.jsx

```diff
@@ -24,17 +23,15 @@ export default class IntentsComponent extends React.Component {
       .post('/mod/nlu/sync')
       .catch(err => this.setState({ syncFailedError: err.response.data }))
       .then(() => {
-        // Set animation timeout
-        setTimeout(() => this.setState({ isSyncing: false }), 2000)
+        this.props.onModuleEvent({ id: 'nlu.complete', working: false, message: 'Model is up-to-date' })
```

Comment:
i'd rather like type instead of id, since id is commonly referred to a uniq identifier